b2

```diff
@@ -801,9 +806,9 @@ def upload_file(args):
             headers['X-Bz-Info-' + k] = b2_url_encode(v)
 
         try:
-            response = post_file(url, headers, local_file, exit_on_error=False, progress_bar=True)
+            response = post_file(url, headers, local_file, exit_on_error=False, progress_bar=(not quiet))
```

Comment:
The extra parenthesis look weird. I would make a new boolean variable `progress_bar` before the `try:` statement to make it easier to read. It doesn't matter very much, we'll clean it up next week anyway.